machine-learning/sagemaker-edge-manager/artifacts/com.greengrass.SageMakerEdgeManager.ObjectDetection/1.0.0/object_detection/LICENSE

```diff
@@ -0,0 +1 @@
+https://greengrass-release-license.s3.us-west-2.amazonaws.com/greengrass-license-v1.pdf
```

Comment:
Our license for these must be a proper open source license (Apache 2), not the v1 license.